Os/Stub/ConditionVariable.cpp

```diff
@@ -9,10 +9,11 @@ namespace Os {
 namespace Stub {
 namespace Mutex {
 
-void StubConditionVariable::wait(Os::Mutex& mutex) {
+StubConditionVariable::Status StubConditionVariable::pend(Os::Mutex& mutex) {
     // This stub implementation can only be used in deployments that never need to wait on any ConditionVariable.
     // Trigger an assertion if anyone ever tries to wait.
     FW_ASSERT(0);
```

Comment:
Do we need this ASSERT anymore?

Reply:
I don't think so. We can just return an error and rely on the calling code to decide for itself what to do. Removing it...